ahk/gui.py

```diff
@@ -0,0 +1,119 @@
+from ahk.script import ScriptEngine
+
+
+class GUIMixin(ScriptEngine):
+
+    TRAYTIP_INFO = 1
+    TRAYTIP_WARNING = 2
+    TRAYTIP_ERROR = 3
+
+    def __init__(self, *args, **kwargs):
+        self.window_encoding = kwargs.pop('window_encoding', None)
+        super().__init__(*args, **kwargs)
+
+    def show_tooltip(self, text: str, second=1.0, x="", y="", id="", blocking=True):
+        """Show ToolTip
+
+        https://www.autohotkey.com/docs/commands/ToolTip.htm
+
+        :param text: Tooltip text
+        :type text: str
+        :param ms: Wait time (s), defaults to 1000
+        :type ms: int, optional
+        :param x: X coordinate relative to active window, defaults to ""
+        :type x: str, optional
+        :param y: Y coordinate relative to active window, defaults to ""
+        :type y: str, optional
+        :param id: ID of ToolTip for more ToolTip message, defaults to ""
+        :type id: str, optional
+        :raises ValueError: ID must be between [1, 20]
+        """
+
+        if id and not (1 <= int(id) <= 20):
+            raise ValueError("ID value must be between [1, 20]")
+
+        script = self.render_template("gui/tooltip.ahk", text=text, second=second, x=x, y=y, id=id)
+        self.run_script(script, blocking=blocking)
+
+    def _show_traytip(self, title: str, text: str, second=1.0, type_id=1, slient=False, large_icon=False, blocking=True):
+        """Show TrayTip (Windows 10 toast notification)
+
+        https://www.autohotkey.com/docs/commands/TrayTip.htm
+
+        :param title: Title of notification
+        :type title: str
+        :param text: Content of notification
+        :type text: str
+        :param second: Wait time (s) to be disappeared, defaults to 1.0
+        :type second: float, optional
+        :param type_id: Notification type `TRAYTIP_<type>`, defaults to 1
+        :type type_id: int, optional
+        :param slient: Shows toast without sound, defaults to False
+        :type slient: bool, optional
+        :param large_icon: Shows toast with large icon, defaults to False
+        :type large_icon: bool, optional
+        """
+
+        option = type_id + (16 if slient else 0) + (32 if large_icon else 0)
+        script = self.render_template("gui/traytip.ahk", title=title, text=text, second=second, option=option)
+        self.run_script(script, blocking=blocking)
```

Comment:
Can not show "utf-8" text like `şğüı` need help @spyoungtech

Reply:
I believe this is a known limitation in several areas and has to do with how encoding is interpreted when sending the script via stdin, which may not be the same across systems. I have some theories on how AHK handles non-ascii input, but I haven't looked into much or confirmed; see #45 

Currently, the best-supported method for non-ascii is to send along AHK-unicode sequences like `{U+2F72}`. Can you give that a try and see if it works?

For example `şğüı` might best be sent as `{u+015f}{u+011f}{u+00fc}{u+0131}`

Reply:
Sorry but I didn't work 🤕

Reply:
I found some way to do this. When I send like this, it's work 🎉
```py
ahk.show_tooltip('% Chr(0x1F680)') # Rocket 🚀
```
![image](https://user-images.githubusercontent.com/49655146/87862463-1b914900-c959-11ea-9844-ebd53908937a.png)

Reply:
Finally, I found usable solution:

```py
from ahk import AHK

ahk = AHK()

string = 'Rocket 🚀 man'
ahk_string = "% "
for char in string:
    hval = hex(ord(char))
    ahk_string += f"Chr({hval})"

ahk.show_tooltip(ahk_string)
```

![image](https://user-images.githubusercontent.com/49655146/87862821-dc64f700-c95c-11ea-8f19-a9fda4d8cf45.png)